docs/setup/migration.md

```diff
@@ -8,11 +8,10 @@ Features that are provided to support this include:
 
 - Ability to import existing password hashes from Synapse
 - Ability to import existing sessions and devices
-- Ability to import existing access tokens
+- Ability to import existing access tokens linked to devices (ie not including short-lived admin puppeted access tokens)
 - Ability to import existing upstream IdP subject ID mappings
 - Provides a compatibility layer for legacy Matrix authentication
 
-If
```

Comment:
Random stray `If` removed while touching the file. Possibly there was a sentence being written here at some point?

Reply:
Probably :D

I think I did notice it at some point, told Hugh about it and we both forgot to remove it 🙈